analytics/components/Plausible.tsx

```diff
@@ -0,0 +1,104 @@
+
+const plausibleScript = exclusionScript;
+
+const sendEvent = (
+  _action: string,
+  _type: string,
+  event: { name?: string; params?: Record<string, string> },
+) => {
+  const origEvent = event?.name;
+  const ecommerce = event?.params;
+
+  if (origEvent && ecommerce) {
+    const flagsObject = {} as Record<string, boolean>;
+    if (window.LIVE.flags) {
+      for (const flag of window.LIVE.flags) {
+        if (flag.name && flag.value) {
+          flagsObject[flag.name] = flag.value;
+        }
+      }
+    }
+    const values = {} as Record<string, string>;
+    for (const key in ecommerce) {
+      if (ecommerce[key] !== null && ecommerce[key] !== undefined) {
+        values[key] = (typeof ecommerce[key] !== "object")
+          ? ecommerce[key]
+          : JSON.stringify(ecommerce[key]);
+      }
+    }
+    window.plausible(origEvent, { props: { ...flagsObject, ...values } });
+  }
+};
+
+function Component({
+  async,
+}: Props) {
+  if (!context.isDeploy) {
+    return <></>;
+  }
+  return (
+    <>
+      <Head>
+        <link rel="dns-prefetch" href="https://plausible.io/api/event" />
+        <link
+          rel="preconnect"
+          href="https://plausible.io/api/event"
+          crossOrigin="anonymous"
+        />
+        {async &&
+          (
+            <Script
+              type="module"
+              data-exclude="/proxy"
+              data-api="https://plausible.io/api/event"
+              dangerouslySetInnerHTML={{
+                __html: plausibleScript,
+              }}
+            />
+          )}
+        {!async && (
+          <script
+            defer
```

Comment:
```suggestion
            type="module"
```
defer doesn't work withou src https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#defer ![image](https://github.com/deco-cx/apps/assets/15680320/947cd457-0a62-4815-9cb5-6c5b12dff5d9)